uiowa/uiowa
# Added remaining background color options to banner block

# How to test

This pr adds the remaining background patterns to the banner block. 


`ddev blt ds --site=sandbox.uiowa.edu`
- Test out patterns on https://sandbox.uiowa.ddev.site/banners-and-buttons
- Verify that patterns appear in black, gold, gray, and white order
- Edit the page and try out all the combinations and verify that they work
- Verify that no update hook is needed for this
- Verify that no other site splits need to be updated

Takes the banner block from:

<img width="910" alt="Screenshot 2023-04-04 at 9 05 16 AM" src="https://user-images.githubusercontent.com/1036433/229818342-447029b6-ccb8-4cb5-a55e-b934fa672980.png">

To

<img width="484" alt="Screenshot 2023-04-04 at 9 04 45 AM" src="https://user-images.githubusercontent.com/1036433/229818381-676ec330-bdab-41de-a33d-4d88b46e0885.png">
